tools/make_vs.bat

```diff
@@ -106,20 +106,33 @@ ninja -C "%REPO_ROOT%%FOLDER%" %TARGET%
     @IF DEFINED VCVARS_x86 IF DEFINED VCVARS_x64 exit /B 0
 
     @REM Trying to find the newest Visual Studio at the default install path
-    @set VC_PREFIX=C:\Program Files (x86)\Microsoft Visual Studio\
-    @IF EXIST "%VC_PREFIX%2019" (
-        set VC_YEAR=2019\
-    ) ELSE IF EXIST "%VC_PREFIX%2017" (
-        set VC_YEAR=2017\
-    ) ELSE exit /B 1
+    @set VC_PREFIX=C:\Program Files\Microsoft Visual Studio\
+    @IF EXIST "%VC_PREFIX%2022" (
+        set VC_YEAR=2022\
+    ) ELSE (
+        echo "Can not find '%VC_PREFIX%2022'"
+        @set VC_PREFIX=C:\Program Files (x86^^^)\Microsoft Visual Studio\
```

Comment:
@Finii:
```suggestion
        @set VC_PREFIX=C:\Program Files (x86)\Microsoft Visual Studio\
```
... or are these characters in some way intentional?

Reply:
Without the escape the closing parens will close the indicated opening one

![image](https://github.com/gul-cpp/gul14/assets/16012374/ba002de3-008d-490c-bca5-ed621639ff27)

Reply:
:+1: Here's me hating batch files.